src/bump.ts

```diff
@@ -897,6 +901,12 @@ export async function bumpSdkVer(
     }
   }
   core.setOutput("next-version", nextVersion?.toString() ?? "");
+  const latestDraftName = latestDraft?.name ?? "";
+  core.setOutput(
+    "previous-commit",
+    latestDraftName.substring(latestDraftName.lastIndexOf(".") + 1)
```

Comment:
I'd prefer using `latestDraft.target_commitish` here, but for that I need to change what `getRelease` returns, because it's not actually a release. This is probably good enough with minimal impact.

Reply:
This file is due some functional and architectural rewrites, but let's not add more implicit and obscure undocumented internal dependencies.

Feel free to do anything other than adding this dependency on the hash being in the name.
Options include modifying `getRelease()` (preferred) or defining a new function to look up the commitish from a release ID in `github.ts`.